pkg/bff/gds.go

```diff
@@ -250,6 +250,12 @@ func (s *Server) SaveRegisterForm(c *gin.Context) {
 		return
 	}
 
+	// Mark the form as started
+	// NOTE: If an empty form was passed in, the form will not be marked as started.
+	if form.State != nil && form.State.Started == "" {
+		form.State.Started = time.Now().Format(time.RFC3339)
+	}
+
```

Comment:
If an empty form is passed in we treat it as a delete since it completely overwrites the existing value in the database. So, I think the right thing to do is not mark the form as started in that case, which is what this code is doing.